packages/@expo/cli/src/start/server/type-generation/routes.ts

```diff
@@ -368,7 +368,7 @@ export function extrapolateGroupRoutes(
  */
 const routerDotTSTemplate = unsafeTemplate`/* eslint-disable @typescript-eslint/no-unused-vars */
 /* eslint-disable import/export */
-/* eslint-disable @typescript-eslint/ban-types */
+/* eslint-disable @typescript-eslint/no-restricted-types */
```

Comment:
I wasn't really aware we were doing that, but I believe we should just add `/* eslint-disable */` and disable it entirely for the template file. I don't see a reason we should keep any checks around on a generated file

cc @Ubax

Reply:
I agree too. https://github.com/expo/expo/pull/41637/changes/7c832e2d52d31b3e06f445e2adc37fbfcca3c1df